demo/demo.html

```diff
@@ -4,29 +4,72 @@
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
     <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
+    <!-- <script src="./temp/sfx-all-plugins-bundle.js"></script> -->
   </head>
+
   <body>
-    <sfx-search-box placeholder="search" clearbutton searchbutton></sfx-search-box>
-    <sfx-sayt visible></sfx-sayt>
-    <sfx-list class="product-grid"></sfx-list>
+    <sfx-search-box id="main-search" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="main-search" visible></sfx-sayt>
+
+    <sfx-search-box id="secondary-search" group="secondary" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="secondary-search" group="secondary" visible></sfx-sayt>
+
+    <sfx-products></sfx-products>
```

Comment:
Since you have duplicated the searchbox and sayt, is it worth duplicating the products grid as well? Not required at all.

Reply:
As it is, one of the searches populates the grid and the other does not. That's valuable to me. This is a reasonable idea, though.